deploy/pom.xml

```diff
@@ -79,7 +80,7 @@
 
 	<profiles>
 		<profile>
-			<id>geowave-tools-singlejar</id>
+			<id>all-deps</id>
```

Comment:
not exactly sure where this came from - I think its a remnant of a previous way services were deployed via the RestServer main method, but Richard's war deployment should make the changes to this file OBE from my understanding.